exhibitor-appliance.yaml

```diff
@@ -19,6 +19,12 @@ SenzaInfo:
     - CustomS3BucketRegion:
         Description: "Use custom s3 bucket region for exhibitor configuration"
         Default: ""
+    - CustomTransactionsDirectory:
+        Description: "Use custom transactions directory"
+        Default: "/opt/exhibitor/transactions"
```

Comment:
@CyberDem0n I added some complementary directory creation commands to the `run.sh` file as well, so to be sure that these directories are going to be created with the right permissions. Lines [17](https://github.com/zalando-incubator/exhibitor-appliance/pull/31/files#diff-1b0c2b516b83393edb7200ad5ff12181R17) and [21](https://github.com/zalando-incubator/exhibitor-appliance/pull/31/files#diff-1b0c2b516b83393edb7200ad5ff12181R21).

I'm not sure if this is the best approach though.